### ISSUE TITLE
 re-encoding procedure hangs when using the example video

### ISSUE THREAD

**Issue by matthewpiper** [groakat/videotagger#102]
(same as what Oisin reported) 
I aborted the copying at step 0/4 after 15min. I then went to the file path and loaded the video, which seemed to display fine and had the bounding boxes you indicate.

**Comment by macaodha** [comment]
I have updated to 1.1.1 and it still happens. Its hangs on step 0/4 even though everything has been created and the last line on the command line is path_to_op_vids/framedataVis framedataVis

**Comment by groakat** [comment]
Hmm. What are you doing exactly? Are you pointing the input folder to a folder with only new and not-yet reencoded files? If you point it to a folder with a single file, the process dialog should only display 0/1 and then disappear after.

If you are using the example project file, you do not need to reencode anything as they are already in `test_video/vt_project/Birds\ flying over the Lake Lugano-HD`

**Comment by macaodha** [comment]
Yes, I'm pointing to the example video and specifying an output folder that
is empty. When I remove everything except the video I want to encode from
the input folder (e.g. the .txt file that tells you about the video source)
there are just 2 steps now, not 4. However it still hangs at the end even
though everythign is done.

**Comment by groakat** [comment]
OK, I see. I fixed it now in the OSX and Linux builds

**Comment by macaodha** [comment]
Works for me now! Thanks
